API_Guidelines.md

```diff
@@ -97,6 +97,31 @@ Payload
 }
 ```
 
+## POST
+
+### Save Method
+
+`Ember.save` will make a POST request *only* if the record is *new*.  
+`Ember.createRecord` will also make a POST request.
```

Comment:
createRecord only creates a record in the local store. it never makes a network req so leave this out altogether since it's covered in the ember docs, this is more about how to create and use apis